deepspeed/module_inject/containers/internlm.py

```diff
@@ -0,0 +1,183 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+# DeepSpeed Team
+
+# Copyright (c) wangruohui
```

Comment:
Hi @wangruohui,

We'd like to kindly ask you to remove this `Copyright (c) wangruohui` line to be in agreement with our [DeepSpeed CLA](https://github.com/microsoft/DeepSpeed#contributor-license-agreement). We value all DeepSpeed contributors and list them in our main [README](https://github.com/microsoft/DeepSpeed#contributing).

Reply:
That's OK. I have removed that line.